C702.tex

```diff
@@ -0,0 +1,160 @@
+%\ssindex{observatories!ground-based!Rubin}
+
+% These lines show examples of ASCL index entries. At this stage these have to commented
+% out, and need to be on separate lines. Eventually, they will be automatically uncommented
+% and used to generate entries in the ASCL Index at the end of the Proceedings volume.
+% The ascl.py command will scan your paper on possible code names.
+% Don't leave these in! - replace them with ones relevant to your paper.
+%ooindex{FOOBAR, ascl:1101.010}
+
+\section{Introduction}
+
+Tabular data catalogs are a fundamental part of modern astronomical research, and relational databases are a commonly used technology for providing access to them, typically via SQL queries.
+Such systems rely on Data Definition Language (DDL) for defining the structure, or schema of the catalog, but DDL lacks the ability to define catalog metadata such as units of measurement or relationships between columns.
+This metadata is critical for understanding and processing the data, as well as presenting it to users in a meaningful way.
+
+The Data Management team at the Vera C.\ Rubin Observatory \citep{2019ApJ...873..111I} has developed a data format for defining the metadata for a particular data catalog or schema.
+Felis\footnote{\url{https://github.com/lsst/felis}} reads and validates this format using the Pydantic Python library\footnote{\url{https://pydantic.dev}}.
+The validated model is used to generate DDL statements or populate a TAP\_SCHEMA database, which can then be accessed through a Table Access Protocol (TAP) service endpoint \citep{2019ivoa.spec.0927D}.
+Felis may also be used for a variety of additional tasks, such as generating documentation or validating schema updates within continuous integration (CI) processes.
+
+\section{Schema Data Model}
+
+\articlefigure{C702_f1.eps}{fig:schema}{ERD diagram of the schema data model.}
+
+All objects in the schema have a set of common attributes, including a name, identifier, and description.
+The name attribute is required and corresponds to the name of the object in the target database.
+The description is an optional, human-readable description of the object.
+The identifier provides a way to uniquely identify the object within the schema and is primarily used for object referencing.
+Identifiers may be generated automatically or provided explicitly by the user.
+
+A version attribute indicates the version of the schema and can be used to track changes over time.
+In addition to the version string, lists of compatible and read-compatible versions can also be provided.
+Felis does not enforce any particular versioning scheme, but semantic versioning is recommended.
+
+A schema must define at least one table, and each table must include a list of one or more columns.
+Columns must have a \texttt{datatype} field and may include a number of optional properties which are used for a variety of purposes, including specifying semantics of the column, such as units, UCD, and xtype.
```

Comment:
Maybe say a couple of words for the uninitiated on what UCD and xtype mean?

Reply:
What about providing references for these instead? (Maybe it should be both.)

Reply:
I spelled out the UCD acronym and provided a footnote with a link the IVOA REC.

Reply:
It should be a formal reference and not a footnote.

Reply:
I wish they provided the references on the IVOA site. 

Does this seem right?

```
@article{article,
author = {Derriere, S. and Gray, Norman and McDowell, J. and Mann, R. and Ochsenbein, Francois and Osuna, Pedro and Martinez, A. and Rixon, G. and Williams, Roy},
year = {2004},
month = {06},
pages = {315},
title = {UCD in the IVOA context},
volume = {314}
}
```

Reply:
I have no idea what the formal reference for the UCD REC is...

Reply:
It's here: https://ui.adsabs.harvard.edu/abs/2023ivoa.spec.0125C/abstract

Includes a DOI (which you can convert to a URL so it displays).

Reply:
ie
```
@MISC{2023ivoa.spec.0125C,
       author = {{Cecconi}, Baptiste and {Louys}, Mireille and {Preite Martinez}, Andrea and {Derri{\`e}re}, S{\'e}bastien and {Ochsenbein}, Fran{\c{c}}ois and {Erard}, St{\'e}phane and {Demleitner}, Markus},
        title = "{UCD1+ controlled vocabulary - Updated List of Terms Version 1.5 Version 1.5}",
 howpublished = {IVOA Endorsed Note 25 January  2023},
         year = 2023,
        month = jan,
        pages = {125},
          url = {https://doi.org/10.5479/ADS/bib/2023ivoa.spec.0125C},
       adsurl = {https://ui.adsabs.harvard.edu/abs/2023ivoa.spec.0125C},
      adsnote = {Provided by the SAO/NASA Astrophysics Data System}
}
```

Reply:
Thank you, @timj. 

I had written my own bibtex entry for UCD, but yours looks better so I'll use it instead.